datahub-web-react/src/app/lineage/LineageTimeSelector.tsx

```diff
@@ -66,9 +67,13 @@ export default function LineageTimeSelector({ onChange, initialDates }: Props) {
                             'Last 7 days': [moment().subtract(7, 'days'), moment()],
                             'Last 14 days': [moment().subtract(14, 'days'), moment()],
                             'Last 28 days': [moment().subtract(28, 'days'), moment()],
-                            'All Time': [moment(0), moment()],
+                            'All Time': [moment(0), moment(0)],
                         }}
                         onChange={(dates, dateStrings) => {
+                            if (dates) {
+                                setStartDate(dates[0]);
```

Comment:
curious why we need this?

Reply:
This is to fix a bug where the date header text would not change because the component would not rerender if all time was selected and would instead say "Until _ date"